tests/client/amo/components/TestAddonDetail.js

```diff
@@ -16,23 +15,26 @@ import { fakeAddon } from 'tests/client/amo/helpers';
 import { getFakeI18nInst } from 'tests/client/helpers';
 
 
-function render({ addon = fakeAddon, ...customProps } = {}) {
+function render({ addon = fakeAddon, setCurrentStatus = sinon.spy(), ...customProps } = {}) {
   const i18n = getFakeI18nInst();
   const initialState = { api: { clientApp: 'android', lang: 'pt' } };
   const props = {
+    ...addon,
```

Comment:
The spread (`...addon`) is added after `addon` above (https://github.com/mozilla/addons-frontend/pull/1388/files#diff-d736a30c9a5a02800a9c3f44a64734bdR27)

Not sure if that matters, just noticed it.